booking/src/main/java/org/dcsa/conformance/standards/booking/checks/AbstractCarrierPayloadConformanceCheck.java

```diff
@@ -18,19 +18,12 @@ abstract class AbstractCarrierPayloadConformanceCheck extends PayloadContentConf
     BookingState.PENDING_AMENDMENT
   );
```

Comment:
Is this class (and any of its subclasses) still used? Based on the changes, I would not be surprised if it was now redundant.

(If it is still used, leave it for now. Ideally, we should migrate all of them to the new BookingChecks method. However, this PR is large enough as it is, so I am fine with the move happening later if it was not already done).